cms/djangoapps/contentstore/tasks.py

```diff
@@ -276,11 +287,11 @@ def export_olx(self, user_id, course_key_string, language):
         user = User.objects.get(pk=user_id)
     except User.DoesNotExist:
         with translation_language(language):
-            self.status.fail(_('Unknown User ID: {0}').format(user_id))
```

Comment:
NIT: you are also updating the export task so the docstring for the `errors.py` can't be just specific to course import.